transport/nio/src/test/java/org/kaazing/gateway/transport/udp/specification/UdpConnectorIT.java

```diff
@@ -166,15 +177,52 @@ protected void doMessageReceived(IoSessionEx session, Object message) {
                     assertEquals("server data 1", decoded);
                     writeStringMessageToSession("client data 2", session);
                     first = false;
+
+                    // FIXME assert here does not seem to make the test fail
+                    assertTrue(false);
                 } else {
                     assertEquals("server data 2", decoded);
+
+                    // FIXME assert here does not seem to make the test fail
+                    assertTrue(false);
                 }
             }
         });
 
         k3po.finish();
     }
 
+
+    @Test
+    @Specification("additions/align.content/server")
+    public void alignData() throws Exception {
+        ResourceAddressFactory addressFactory = ResourceAddressFactory.newResourceAddressFactory();
+        ResourceOptions resourceOptions = ResourceOptions.FACTORY.newResourceOptions();
+        resourceOptions.setOption(UdpResourceAddress.ALIGN, 4);
+        ResourceAddress connectAddress = addressFactory.newResourceAddress("udp://127.0.0.1:8080", resourceOptions);
+        AtomicBoolean bytesAligned = new AtomicBoolean(true);
+        CountDownLatch messagesReceived = new CountDownLatch(2);
+        ConnectFuture connectFuture = connector.connect(connectAddress, new IoHandlerAdapter<IoSessionEx>(){
```

Comment:
Yes, I'm checking the message length is a multiple of 4 ` bytesAligned.set(((IoBuffer) message).remaining() % 4 == 0);`